src/boot/test/parse-json-config-test.js

```diff
@@ -0,0 +1,139 @@
+import { parseJsonConfig } from '../parse-json-config';
```

Comment:
Note for self: these tests were lifted from the (now-deleted) `settings-test` module.

Question for @robertknight Any meaningful changes from that set of tests?

Reply:
No - the tests are the same as before except for the name change.